### PR TITLE
Ahmad main

### PR DESCRIPTION
**Pull request by amoughnie**
Added Habit.java class

### CODE REVIEW

File: app/src/main/java/com/example/ayapp/habits/Habit.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.example.ayapp.habits;
+
+/**
+ * Created by amoughnie on 07-May-16.
+ */
+public class Habit {
+
+    private String title;// habit title
```
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java]: No need for inline comments if the variable is clear.

File: app/src/main/java/com/example/ayapp/habits/Habit.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public Habit(){
+        this.title = null;
+        this.description = null;
+        this.goal = 0;
+        this.id = 0;
+    }
+
+
```
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java]: Extra line here.

File: app/src/main/java/com/example/ayapp/habits/Habit.java (hunk 1 above)
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java]: It's ok though, it's just a remark for next time :)

File: app/src/main/java/com/example/ayapp/habits/Habit.java (hunk 2 above)
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java]: Here you can add a section comment like this:

/***********************************************************************************************
     \* SETTERS AND GETTERS
     **********************************************************************************************/

File: app/src/main/java/com/example/ayapp/habits/Habit.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    private int id; // used as an index in the database
+
+    public Habit(String title,String description, int goal, int id){
+        this.title = title;
+        this.description = description;
+        this.goal = goal;
+        this.id = id;
+    }
+
+    public Habit(){
```
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java:20]: Please put the default constructor above the regular constructor and add a section comment:

/***********************************************************************************************
     \* CONSTRUCTORS
     **********************************************************************************************/

File: app/src/main/java/com/example/ayapp/habits/Habit.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
+package com.example.ayapp.habits;
+
+/**
+ * Created by amoughnie on 07-May-16.
+ */
+public class Habit {
+
+    private String title;// habit title
+    private String description;// habit description
+    private int goal;// number of days in which you want to preform this habit
```
Comment (TheCodingGent) [inline app/src/main/java/com/example/ayapp/habits/Habit.java:14]: here the comment is good for sure

File: app/src/main/java/com/example/ayapp/habits/Habit.java (hunk 1 above)
Comment (amoughnie) [inline app/src/main/java/com/example/ayapp/habits/Habit.java]: Acknowledged

**Pull request merged**